cmd/ui/v1alpha2/main.go

```diff
@@ -0,0 +1,37 @@
+package main
+
+import (
+	"net/http"
+
+	ui "github.com/kubeflow/katib/pkg/ui/v1alpha2"
+)
+
+var (
+	port = "80"
+)
+
+func main() {
+	kuh := ui.NewKatibUIHandler()
+
+	frontend := http.FileServer(http.Dir("/app/build/"))
+	http.Handle("/katib/", http.StripPrefix("/katib/", frontend))
```

Comment:
It will be one deployment with frontend and backend. 
All JS for frontend will be store in `build` folder here: https://github.com/andreyvelich/katib/tree/v1alpha2-new-ui/pkg/ui/v1alpha2/frontend/build/static.

Reply:
I wonder if the container works too heavy.

Reply:
I think it will be fine, no reason to divide frontend and backend sides. 
All frontend builds in a 3 JS files by React.